Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -40,85 +42,86 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   }
 
   using TestImageType = itk::Image<float, 3>;
-  auto reader_lambda = [](std::string filename) -> TestImageType::Pointer {
+  auto ReaderLambda = [](std::string filename) -> TestImageType::Pointer {
```

Comment:
This lambda is no longer necessary (see below).

Reply:
Thanks for this, I replaced it

Reply:
:+1: Done in ceac451.